DHL/Datatype/GB/RegistrationNumbers.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Note : Code is released under the GNU LGPL
+ *
+ * Please do not change the header of this file
+ *
+ * This library is free software; you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation; either version 2 of
+ * the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU Lesser General Public License for more details.
+ */
+
+/**
+ * File:        RegistrationNumbers.php
+ * Project:     DHL API
+ *
+ * @author      Al-Fallouji Bashar
+ * @version     0.1
+ */
+
+namespace DHL\Datatype\GB;
+use DHL\Datatype\Base;
+
+/**
+ * RegistrationNumbers Request model for DHL API
+ */
+class RegistrationNumbers extends Base
```

Comment:
I think you don't need that intermediate object and can just specify something like (leaving `disableParentNode` to default - false)
```
        'RegistrationNumbers' => array(
            'type' => 'RegistrationNumber',
            'required' => false,
            'subobject' => true,
            'multivalues' => true,
        ),
```
Like with Pieces in `DHL/Datatype/AM/BkgDetailsType.php:115`

Or am I missing something?